src/base-element.js

```diff
@@ -82,9 +84,18 @@ import {vsyncFor} from './vsync';
  * before buildCallback and it might be called multiple times including
  * after layoutCallback.
  *
- * The documentInactiveCallback is called when the document becomes
- * inactive. E.g. when the user swipes away from the document or
- * focuses a different tab.
+ * The pauseCallback is called when when the document becomes inactive, e.g.
+ * when the user swipes away from the document. In these situations, any
```

Comment:
Please document here or in the diagram what is the relationship with layout. Can it be paused before layout? After?